pkgs/tools/security/mkpasswd/default.nix

```diff
@@ -16,6 +16,5 @@ stdenv.mkDerivation {
     description = "Overfeatured front-end to crypt, from the Debian whois package";
     license = licenses.gpl2;
     maintainers = with maintainers; [ cstrahan fpletz ];
-    platforms = platforms.linux;
```

Comment:
please don't delete `platforms`, since it needs to be explicit i.e. `platforms = platforms.unix;` otherwise hydra won't build it

Reply:
Thanks, fixed.